cg/models/demultiplex/flow_cell.py

```diff
@@ -204,21 +226,10 @@ def validate_sample_sheet(self) -> bool:
 
     def get_sample_sheet(self) -> SampleSheet:
         """Return sample sheet object."""
-        try:
-            return get_sample_sheet_from_file(
-                infile=self.sample_sheet_path,
-                flow_cell_sample_type=self.sample_type,
-            )
-        except Exception as error:
-            alternative_sample_sheet_path: Path = Path(
-                self.path,
-                DemultiplexingDirsAndFiles.UNALIGNED_DIR_NAME,
-                DemultiplexingDirsAndFiles.SAMPLE_SHEET_FILE_NAME,
-            )
-            return get_sample_sheet_from_file(
-                infile=alternative_sample_sheet_path,
-                flow_cell_sample_type=self.sample_type,
-            )
+        return get_sample_sheet_from_file(
+            infile=self.sample_sheet_path,
+            flow_cell_sample_type=self.sample_type,
+        )
```

Comment:
See above regarding error handling would be nice to have some indication if it fails

Reply:
There is a warning logged in the post processing flow when a fastq file cannot be processed so I think it is fine for now